CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/server/RequestHandler/FunctionRequestHandler.java

```diff
@@ -0,0 +1,229 @@
+package com.couchbase.CouchbaseLiteServ.server.RequestHandler;
```

Comment:
Do we need this class, as we don't have any test for them?

Reply:
no need to cover tests in API? Raghu?

Reply:
Its ok to leave it in there as its already written. Might be needed in the future.